ecc/Dockerfile.boilerplate-ecc

```diff
@@ -13,7 +13,8 @@ RUN mkdir -p ${CC_LIB_PATH}
 ENV SGX_SDK=/opt/intel/sgxsdk
 ENV LD_LIBRARY_PATH=${SGX_SDK}/sdk_libs:${CC_LIB_PATH}
 
-copy ecc/${CC_NAME} ${CC_PATH}/chaincode
-copy ecc_enclave/_build/lib/*.so ${CC_LIB_PATH}/
+# TODO: the boiler plate should include the compiled ecc
```

Comment:
I thought we wanted to outsource the MRENCLAVE into a text file rather than baking it into ecc (and maybe revisit why what's the point of having MRENCLAVE of fpc chaincode used inside ecc in the first place). 
I guess you dropped that as you run out of time?  If so, we should at least add a corresponding issue (and reference it here)?

Reply:
right, this is one of the improvements I have mentioned above.
I will convert these to issues once this is merged